synapse/lib/stormtypes.py

```diff
@@ -268,20 +455,73 @@ def addLibFuncs(self):
             'guid': self._guid,
             'fire': self._fire,
             'list': self._list,
-            'null': None,
-            'true': True,
-            'false': False,
+            'null': self._null,
+            'true': self._true,
+            'false': self._false,
             'text': self._text,
             'cast': self._cast,
             'warn': self._warn,
             'print': self._print,
             'pprint': self._pprint,
             'sorted': self._sorted,
             'import': self._libBaseImport,
-        })
+        }
+
+    @property
```

Comment:
using @property allows us to get an inspectable Property object during documentation generation; but the value is properly resolved during runtime